diff --git a/includes/flag_plugin_argument_validate_flaggability.inc b/includes/flag_plugin_argument_validate_flaggability.inc
index 1efe9af..0cba7ed 100644
--- a/includes/flag_plugin_argument_validate_flaggability.inc
+++ b/includes/flag_plugin_argument_validate_flaggability.inc
@@ -26,6 +26,11 @@ class flag_plugin_argument_validate_flaggability extends views_plugin_argument_v
   }
 
   function options_form(&$form, &$form_state) {
+    // If there are no flag relationships set, skip this form of validation.
+    $flags = flag_get_flags($this->flag_type);
+    if (!$flags) {
+      return array();
+    }
     $options = $this->flags_options();
 
     $form[$this->_option_name('flag_name')] = array(
