diff --git a/sites/all/modules/userpoints_contrib/userpoints_reset/userpoints_reset.module b/sites/all/modules/userpoints_contrib/userpoints_reset/userpoints_reset.module
index 7d58a41..2d94230 100644
--- a/sites/all/modules/userpoints_contrib/userpoints_reset/userpoints_reset.module
+++ b/sites/all/modules/userpoints_contrib/userpoints_reset/userpoints_reset.module
@@ -1,14 +1,13 @@
 <?php
 
-define(USERPOINTS_PERM_RESET, 'reset userpoints');
+define( 'USERPOINTS_PERM_RESET', 'reset userpoints');
 
 function userpoints_reset_help($path, $arg) {
   switch ($path) {
     case 'admin/settings/userpoints/reset':
       $output = t('Reset userpoints');
-      break;
+      return $output;
   }
-  return $output;
 }
 
 function userpoints_reset_menu() {
@@ -26,7 +25,7 @@ function userpoints_reset_perm() {
 }
 
 function userpoints_reset_page() {
-  return drupal_get_form('userpoints_reset', $form);
+  return drupal_get_form( 'userpoints_reset' );
 }
 
 function userpoints_reset() {
@@ -56,4 +55,5 @@ function userpoints_reset_submit($form, &$form_state) {
 function userpoints_reset_do() {
   db_query('TRUNCATE {userpoints}');
   db_query('TRUNCATE {userpoints_txn}');
+  db_query('TRUNCATE {userpoints_total}');
 }
